downloaded/installLisk.sh

```diff
@@ -39,6 +39,10 @@ if [ "$USER" == "root" ]; then
 fi
 
 prereq_checks() {
+	if [[ $- != *i* ]]; then
```

Comment:
Shouldn't this be an option that can be passed?

Reply:
We'll use the [-n] that I forgot to get rid of in the usage output ;-)